docs/introduction-to-smart-contracts.rst

```diff
@@ -565,3 +565,21 @@ contracts, the Ether is forever lost.
 If you want to deactivate your contracts, you should instead **disable** them
 by changing some internal state which causes all functions to revert. This
 makes it impossible to use the contract, as it returns Ether immediately.
+
+
+.. index:: ! precompiled contracts, precompiles, contract;precompiled
+
+.. _precompiledContracts:
+
+Precompiled Contracts
+=====================
+
+There is a small set of contract addresses that are special:
+The address range between `1` and (including) `8` contain
```

Comment:
Maybe it would be useful to list them? Or at least link somewhere where people can find a list?

Reply:
I thought about it, but I think it goes a bit too much into detail for this introductory section. The solidity documentation is not meant as an evm reference.

Reply:
Fair enough. Personally, I think it would be better to link to them anyway but I can see the logic behind not doing that.